.github/workflows/cypress-test-oidc-e2e.yml

```diff
@@ -129,13 +130,28 @@ jobs:
           opensearch_security.openid.connect_url: "http://127.0.0.1:8080/auth/realms/master/.well-known/openid-configuration"
           opensearch_security.openid.client_id: "opensearch"
           opensearch_security.openid.client_secret: "${{ env.TEST_KEYCLOAK_CLIENT_SECRET }}"
-          opensearch_security.auth.type: ["openid"]
-          opensearch_security.auth.multiple_auth_enabled: true
-          opensearch_security.ui.openid.login.buttonname: "OIDC"
+          opensearch_security.openid.base_redirect_url: http://localhost:5601${{ matrix.basePath }}
+          opensearch_security.auth.type: "openid"
           home.disableWelcomeScreen: true
           EOT
 
+      - name: Run OSD with basePath
+        if: ${{ matrix.basePath != '' }}
+        run: |
+          echo "server.basePath: \"${{ matrix.basePath }}\"" >> opensearch_dashboards_openid.yml
+          echo "server.rewriteBasePath: true" >> opensearch_dashboards_openid.yml
+
+      - name: Run Cypress Tests with basePath
+        if: ${{ matrix.basePath != '' }}
+        uses: ./.github/actions/run-cypress-tests
+        with:
+          security_config_file: config_openid.yml
+          dashboards_config_file: opensearch_dashboards_openid.yml
+          yarn_command: 'yarn cypress:run --browser chrome --headless --spec "test/cypress/e2e/oidc/*.js" --env basePath=${{ matrix.basePath }}'
+          osd_base_path: ${{ matrix.basePath }}
```

Comment:
Same comment as above, can we consolidate these? The run-cypress-tests action should be able to handle empty vs /osd input

Reply:
+1

Reply:
I wasn't able to figure out another way to do this to reduce it to 1 step.